src/Domain/LeanCode.DomainModels.MassTransitRelay/DbContextExtensions.cs

```diff
@@ -8,14 +8,17 @@ public static class DbContextExtensions
         public static string GetFullTableName(this DbContext dbContext, Type entity)
         {
             var type = dbContext.Model.FindEntityType(entity);
-            if (string.IsNullOrEmpty(type.GetSchema()))
```

Comment:
That's a totally unnecessary change. Please revert.

Reply:
It is not, otherwise a dereference of possibly–`null` variable `type` is reported as warning.

Reply:
Yeah, then add the `type is null` guard but don't change the rest - it's unnecessary in this PR.

Nevertheless, let's leave it changed for now.

Reply:
Fair enough, I just couldn't help myself when I saw an opportunity for a refactor that would let us avoid unnecessary method call :see_no_evil:

But the ternary operator… yeah, I shouldn't have done that, not in this PR at least.